CMakeLists.txt

```
@@ -787,12 +795,12 @@ endif()
 
 # Tests #######################################################################
 #
-
-#if(BUILD_TESTING)
-#    enable_testing()
-#
-#    add_test(...)
-#endif()
+if(BUILD_TESTING)
```

Do we have this somewhere?
```suggestion
include(CTest)
if(BUILD_TESTING)
```
https://cmake.org/cmake/help/latest/module/CTest.html

---

Ah, can you comment in line 199 please?

---

Done in [d63c920](https://github.com/ECP-WarpX/WarpX/pull/5068/commits/d63c920d81e25877bfa54d7fa8d36ee6a9379497).